source/_components/pioneer.markdown

```diff
@@ -40,9 +44,84 @@ timeout:
   description: Number of seconds (float) to wait for blocking operations like connect, write and read.
   required: false
   type: float
+sources:
+  description: A list of mappings from source friendly name to source code (e.g. `TV:'05'`). Valid source codes depend on the receiver (some known codes can be found below). Codes must be defined as strings (between single or double quotation marks), so that `05` is not implicitly transformed to `5`, which wouldn't be a valid source code.
```

Comment:
```suggestion
  description: A list of mappings from source friendly name to the source code (e.g. `TV:'05'`). Valid source codes depend on the receiver (some known codes can be found below). Codes must be defined as strings (between single or double quotation marks) so that `05` is not implicitly transformed to `5`, which wouldn't be valid source code.
```